lib/generators/deimos/schema_class_generator.rb

```diff
@@ -42,28 +43,35 @@ def deimos_field_type(schema_field)
         # @param key_schema_name [String]
         def generate_classes(schema_name, namespace, key_schema_name)
           schema_base = Deimos::SchemaBackends::AvroBase.new(schema: schema_name, namespace: namespace)
-          generate_classes_from_schema_base(schema_base)
+          schema_base.load_schema
+          if key_schema_name.present?
+            key_schema_base = Deimos::SchemaBackends::AvroBase.new(schema: key_schema_name, namespace: namespace)
```

Comment:
Is there a reason we have to hard code this to Avro? We should be able to use `Deimos.schema_backend`

Reply:
Yeah - i _validate_ currently to enforce Avro, can change after i update the generator rspec to include
`deimos_config.schema.backend = :avro_schema_registry` or something

Reply:
yeap - this is doable!